### PR TITLE
Solution js_cat_and_dogs_years

### PR DESCRIPTION
**Pull request by Viktoriia17051991**

### CODE REVIEW

File: src/getHumanAge.test.js [hunk 1]
```diff
@@ -3,8 +3,39 @@
 describe('getHumanAge', () => {
   const { getHumanAge } = require('./getHumanAge');
 
-  test('should be declared', () => {
-    expect(getHumanAge)
-      .toBeInstanceOf(Function);
+  test('should return [0, 0] for catAge = 0 and dogAge = 0', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
```
Comment (Ivan-Shvorak) [inline src/getHumanAge.test.js:9]: Please add test for min - 1 (15 - 1 = 14 // 0 years) to check boundaries

File: src/getHumanAge.test.js [hunk 2]
```diff
@@ -3,8 +3,39 @@
...
+  test('should return [1, 1] for catAge = 15 and dogAge = 15', () => {
+    expect(getHumanAge(15, 15)).toEqual([1, 1]);
+  });
+
```
Comment (Ivan-Shvorak) [inline src/getHumanAge.test.js:13]: Please also check 2 years and boundaries for this value

File: src/getHumanAge.test.js [hunk 3]
```diff
@@ -3,8 +3,39 @@
...
+    expect(getHumanAge(28, 29)).toEqual([3, 3]);
+  });
+
+  test('should return [3, 3] for catAge = 30 and dogAge = 30', () => {
+    expect(getHumanAge(30, 30)).toEqual([3, 3]);
+  });
+
+  test('should return [6, 5] for catAge = 40 and dogAge = 40', () => {
+    expect(getHumanAge(40, 40)).toEqual([6, 5]);
+  });
```
Comment (Ivan-Shvorak) [inline src/getHumanAge.test.js:24]: These tests do the same

File: src/getHumanAge.test.js [hunk 4]
```diff
@@ -3,8 +3,39 @@
...
+    expect(getHumanAge(5, 15)).toEqual([0, 1]);
+  });
+
+  test('should return [3, 0] for catAge = 28 and dogAge = 8', () => {
+    expect(getHumanAge(28, 8)).toEqual([3, 0]);
+  });
+
+  test('should return [11, 9] for catAge = 60 and dogAge = 60', () => {
+    expect(getHumanAge(60, 60)).toEqual([11, 9]);
+  });
```
Comment (Ivan-Shvorak) [inline src/getHumanAge.test.js:36]: Please use boundary values techniques

**Review by Ivan-Shvorak — CHANGES_REQUESTED** [review]
Please check the comments

File: src/getHumanAge.test.js (hunk 1 above)
Comment (4ksandrey) [reply src/getHumanAge.test.js:9]: Not fixed, (getHumanAge(14, 14) should return [0, 0] you don't need to verify minus values)

File: src/getHumanAge.test.js (hunk 2 above)
Comment (4ksandrey) [reply src/getHumanAge.test.js:13]: Why did you remove this test?) you need to add more, should be at least all boundaries, for example: getHumanAge(14, 14), getHumanAge(15, 15), getHumanAge(27, 27), getHumanAge(28,28)

**Review by 4ksandrey — CHANGES_REQUESTED** [review]

**Review by Kate-Solovyova — CHANGES_REQUESTED** [review]
Not all tests were added.  What about getHumanAge (100, 100), getHumanAge(23, 23),
getHumanAge(24, 24)... Please, add more

**Review by 4ksandrey — APPROVED** [review]
